src/components/Lists.tsx

```diff
@@ -116,10 +117,10 @@ export function ListHeaderDesktop({
   title: string
   subtitle?: string
 }) {
-  const {gtTablet} = useBreakpoints()
+  const {isDesktop} = useWebMediaQueries()
   const t = useTheme()
 
-  if (!gtTablet) return null
+  if (!isDesktop) return null
```

Comment:
`gtTablet` and `isDesktop` still are not 1:1. The current prod behavior lets you hit a ~15px area where the header just won't show up because the `List` component uses `isDesktop` vs this using `gtTablet`

Reply:
I thought we fixed this. Can we not align them?

Reply:
Fixed in [`f444f40` (#4982)](https://github.com/bluesky-social/social-app/pull/4982/commits/f444f40131447983f7d0ef6a672090442cdb1313)